### PR TITLE
Finalize handling of IPlaceholderOperation in control flow graph.

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@333fred, @mavasani, @cston, @dotnet/roslyn-compiler, @dotnet/analyzer-ioperation Please review

File: src/Compilers/Core/Portable/Operations/OperationVisitor.cs
Comment (jcouv) [inline src/Compilers/Core/Portable/Operations/OperationVisitor.cs]: Can the comment be removed (here and below, at line 709)?

File: src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb
Comment (jcouv) [inline src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb]: Does the `Into` syntax allow for explicit names for the anonymous type elements?

**Review by jcouv — APPROVED** [review]
LGTM

**Comment by AlekseyTs** [comment]
@333fred, @cston, @dotnet/roslyn-compiler Please review, need second review

**Comment by AlekseyTs** [comment]
@mavasani, @dotnet/analyzer-ioperation Please review

File: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs
Comment (agocke) [inline src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs]: Why did this method move?

**Review by agocke — APPROVED** [review]
LGTM

File: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs
Comment (AlekseyTs) [reply src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs]: > Why did this method move?

The region below contains methods that haven't been finalized. Once that happens, we move the method out of it.

**Review by mavasani — APPROVED** [review]

File: src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb
Comment (AlekseyTs) [reply src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb]: > Does the Into syntax allow for explicit names for the anonymous type elements?

Yes it does.

---
In reply to: [192755825](https://github.com/dotnet/roslyn/pull/27391#discussion_r192755825) [](ancestors = 192755825)

Comment (jcouv) [reply src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb]: Would it make sense to test an aggregate with names then?

Comment (AlekseyTs) [reply src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb]: > Would it make sense to test an aggregate with names then?

Names play no role in the control flow or the shape of the nodes we are going after in this PR. Given the time constraints, we are not doing discovery testing.

---
In reply to: [192882767](https://github.com/dotnet/roslyn/pull/27391#discussion_r192882767) [](ancestors = 192882767)

**Pull request merged**
